lib/src/chat/chat_profile_single.dart

```diff
@@ -137,24 +159,57 @@ class _ChatProfileOneToOneState extends State<ChatProfileOneToOne> {
                   },
                 ),
               ),
-            if (!isInvite())
-              ProfileAction(
-                iconData: IconSource.delete,
-                color: CustomTheme.of(context).error,
+              if (!widget.isSelfTalk)
+                ListGroupHeader(
+                  text: L10n.get(L.settingP),
+                ),
+              SettingsItem(
+                icon: IconSource.notifications,
+                text: L10n.get(L.settingItemNotificationsTitle),
+                iconBackground: CustomTheme.of(context).notificationIcon,
+                onTap: () =>
+                    _settingsItemTapped(context, SettingsItemName.notification),
+              ),
+              if (!isInvite())
+                ListGroupHeader(
+                  text: "",
+                ),
+              SettingsItem(
```

Comment:
Shouldn't this be indented like the ListGroupHeader? Otherwise the `if (!isInvite())` won't work and as far as I see the invite check should be applied to the delete ProfileAction.